managed_config/index.bs

```diff
@@ -0,0 +1,116 @@
+<pre class='metadata'>
+Title: Managed Configuration API
+Shortname: managed-configuration
+Level: 1
+Editor: Anatoliy Potapchuk https://www.w3.org/users/130190, Google LLC https://www.google.com, apotapchuk@google.com
+Status: w3c/CG-DRAFT
+Issue Tracking: Github https://github.com/WICG/WebApiDevice/issues
```

Comment:
I think this is redundant now that you have the `Repository:` metadata. In the front matter I now see two GitHub issue tracker links.